src/containers/AboutUsPage/index.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../style/variables";
+
+.aboutUsPage {
+  h2 {
+    color: $primaryColor;
+    font: $mainFontFamily;
```

Comment:
I think this should be font-family not font, same thing for p.

Reply:
will fix